### ISSUE TITLE
closeable alert aria-label for close button missing

### ISSUE THREAD

**Issue by bmakuh** [react-bootstrap/react-bootstrap#1552]
The 'close' button for closeable alerts is missing an `aria-label` attribute, so for screen readers it currently is read as 'times' rather than 'close'.

http://react-bootstrap.github.io/components.html#alerts-closeable

**Comment by jquense** [comment]
that's not right, the `x` close button has `aria-hidden` and is not for screen readers, there is another close button injected into the alert at the end of it for screen readers.

**Comment by bmakuh** [comment]
Well then the issue is that it should have a `tabIndex='-1'` attribute applied, because it is currently part of the tab index.

**Comment by jquense** [comment]
#1545

**Comment by bmakuh** [comment]
Excellent. Thanks, @jquense.
